### PR TITLE
Make subnet source NAT configurable

### PR DESCRIPTION
**Pull request by lmagyar**
# Proposed Changes

Makes possible to fully utilize site-to-site networking, allows to disable source NAT for subnet devices.

Source: https://tailscale.com/kb/1214/site-to-site/ Step 1 / Point 2

### CODE REVIEW

**Comment by lmagyar** [comment]
not stale

**Comment by lmagyar** [comment]
not stale

File: tailscale/DOCS.md [hunk 1]
```diff
@@ -271,6 +272,17 @@ More information: [Enabling HTTPS][tailscale_info_https]
 previously to access Home Assistant. Tailscale Proxy works on the default HTTPS
 port 443._
 
+### Option: `snat_subnet_routes`
+
+This option allows subnet devices to see the traffic originating from the subnet
+router, and this simplifyies routing configuration.
```
Comment (frenck) [inline tailscale/DOCS.md:278]:
```suggestion
router, and this simplifies routing configuration.
```

File: tailscale/translations/en.yaml [hunk 2]
```diff
@@ -50,6 +50,14 @@ configuration:
       This option allows you to enable Tailscale's Proxy feature to present your
       Home Assistant instance on your tailnet with a valid certificate.
       When not set, this option is enabled by default.
+  snat_subnet_routes:
+    name: Source NAT subnet routes
+    description: >-
+      This option allows subnet devices to see the traffic originating from the
+      subnet router, and this simplifyies routing configuration.
```
Comment (frenck) [inline tailscale/translations/en.yaml:57]:
```suggestion
      subnet router, and this simplifies routing configuration.
```

**Review by frenck — APPROVED** [review]
Thanks, @lmagyar 👍 

../Frenck

**Pull request merged**